SCClassLibrary/Common/Core/Kernel.sc

```diff
@@ -473,9 +473,17 @@ FunctionDef {
 	checkCanArchive { "cannot archive FunctionDefs".warn }
 	archiveAsCompileString { ^true }
 
+	hasPartialApplication {
+		^this.numArgs > 0 and: { argNames[0] == \_ }
+	}
```

Comment:
I think this is where the problem is - argNames might be `nil`. Possible solution:

```supercollider
	hasPartialApplication {
		^this.numArgs > 0 and: { argNames.notNil } and: { argNames[0] == \_ }
	}
```

Reply:
also, shouldn't the whole expression be in parentheses? (e.g. `^(this.numArgs > 0 and: {...} )`)